python/py-requests-cache/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        reclosedev requests-cache 0.5.0 v
+github.setup reclosedev requests-cache 6717a336430eb7e6fe17e75c5517f3076ca85b42
+version 0.5.2
```

Comment:
Please indent these lines to match the rest of the lines in the portfile.

Reply:
done